xrpl_api/src/lib.rs

```diff
@@ -1,8 +1,8 @@
 //! A low-level crate that extracts the common RPC protocol used by the HTML
 //! and WebSocket clients.
 
-pub mod api;
-pub mod types;
+mod api;
+mod types;
 
 pub use api::*;
```

Comment:
I am wondering if we could remove the `pub use` instead, thoughts?

Reply:
I think the `pub use` is good in many cases. It means that you don't expose the internal module structure to api consumers. That is generally good.

Reply:
I like it in the `xrpl_api` and `xrpl_types` crates, do you see any downsides?

Reply:
Will not make any changes for the moment.